graphql2/graphqlapp/alert.go

```diff
@@ -155,6 +175,52 @@ func (a *Alert) Service(ctx context.Context, raw *alert.Alert) (*service.Service
 	return (*App)(a).FindOneService(ctx, raw.ServiceID)
 }
 
+func (a *Alert) RecentEvents(ctx context.Context, obj *alert.Alert, opts *graphql2.AlertRecentEventsOptions) (*graphql2.AlertLogEntryConnection, error) {
+	if opts == nil {
+		opts = new(graphql2.AlertRecentEventsOptions)
+	}
+
+	var s alertlog.SearchOptions
+	if opts.After != nil && *opts.After != "" {
+		err := search.ParseCursor(*opts.After, &s)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if opts.Limit != nil {
+		s.Limit = *opts.Limit
+	}
+	if s.Limit == 0 {
+		s.Limit = search.DefaultMaxResults
+	}
+
+	s.FilterAlertIDs = append(s.FilterAlertIDs, obj.ID)
```

Comment:
Search options (other than `Limit` and `After`) should be set before `ParseCursor` otherwise the cursor won't behave properly.